.werft/util/certs.ts

```diff
@@ -91,64 +79,6 @@ function retrieveFailedCertDebug(certName: string, slice: string) {
     );
 }
 
-function validateSubdomains(werft: Werft, slice: string, domain: string, subdomains: string[]): void {
```

Comment:
No need for this one as we cap the preview name to 20 chars and combined with the existing domain we never exceed 63 chars.